types/src/front/assistant/actions/retrieval.ts

```diff
@@ -4,6 +4,7 @@
 
 import { ModelId } from "../../../shared/model_id";
 import { ioTsEnum } from "../../../shared/utils/iots_utils";
+import { BaseAction } from "../../lib/api/assistant/actions";
```

Comment:
Must come from the root I beleive? so `../../../front`

Reply:
cc @lasryaric 

Also is it cool to use index files? This file really is `actions/index.ts`.

Reply:
https://github.com/dust-tt/dust/pull/5110

Reply:
Not sure why I am mentioned but yes we should `../../` up to root, and then go back down the tree.